ocaml/typing/env.ml

```diff
@@ -4294,17 +4114,8 @@ let report_lookup_error _loc env ppf = function
            "but modules are not module types"
     end
   | Unbound_cltype lid ->
-<<<<<<< HEAD
       fprintf ppf "Unbound class type %a" !print_longident lid;
       spellcheck ppf extract_cltypes env lid
-||||||| 121bedcfd2
-      fprintf ppf "Unbound class type %a" !print_longident lid;
-      spellcheck ppf extract_cltypes env lid;
-=======
-      fprintf ppf "Unbound class type %a"
-        (Style.as_inline_code !print_longident) lid;
```

Comment:
Is there a reason not to take the `Style.as_inline_code` here?

Reply:
Nope, that was a mistake. Updating.